src/normalizeBy.js

```diff
@@ -1,13 +1,13 @@
-const assoc  = require('ramda/src/assoc')
-const curry  = require('ramda/src/curry')
-const reduce = require('ramda/src/reduce')
+const assoc    = require('ramda/src/assoc')
+const reduce   = require('ramda/src/reduce')
+const uncurryN = require('ramda/src/uncurryN')
 
 // normalizeBy :: String -> [{ k: v }] -> { v: { k: v } }
-const normalizeBy = (id, list) =>
-  reduce(putBy(id), {}, list)
+const normalizeBy = id =>
+  reduce(putBy(id), {})
```

Comment:
Do we maybe want to deprecate `normalizeBy`, since Ramda has an `R.indexBy`?  Or maybe reimplement the former in terms of the latter?  I definitely didn't know about `R.indexBy` when I originally wrote this.

Reply:
`normalizeBy('id')` ~= `indexBy(prop('id'))`. I think the shorter contract has value.